CONTRIBUTING.md

```diff
@@ -26,10 +32,26 @@ The last option is needed because our guides feature code examples. **markdown-s
 
 For now it'll simply report all errors and you can then deal with actually spelling errors directly.
 
-We use [markdownlint](https://github.com/mivok/markdownlint) to check the files have a consistent style as part of a CI process. We check contributions using `mdl --style md_style.rb $PWD`. We recommend you install this and run it before pushing your commits.
+We use [markdownlint](https://github.com/mivok/markdownlint) to check the files have a consistent style as part of a CI process. We check contributions using `mdl --style md_style.rb $PWD`. We recommend you install this and run it before requesting feedback.
+
+## When you're finished
+
+Make sure everything is pushed, then add the **ready for review** label to the PR and ensure you are assigned. If you have access to it announce your PR in our [Slack](https://defra-digital.slack.com/) [#dst-guides](https://defra-digital.slack.com/messages/dst-guides/) channel.
+
+If you don't add a comment mentioning either **@cruikshanks** or **@bensagar-ea**.
 
 ## Getting feedback
 
-Feedback will be provided by the team within the PR. Decisions will be documented and reasons given.
+Feedback will be provided within the PR. Decisions will be documented and reasons given.
+
+We want to encourage all to contribute and suggest ways we can improve. Anyone providing feedback should be positive and professional with any comments made about your contribution, but let us know if you feel we failed.
+
+We don't expect everyone to be available or able to contribute but we want to allow enough time for those who wish to get involved. Don't merge your PR for at least 3 days, even if you've had lots of positive feedback.
+
+## Finishing up
+
+Leave the PR open for at least 3 working days and ensure at least one other person has reviewed it.
+
+Once those who have participated in the PR have said they are happy, first [squash]() your commits down to a single commit (checkout the **Completing the commit** section in [Pull requests](/process/pull_request.md) if you are unsure how to do this). Then merge it in GitHub (not locally). Don't forget to also delete the branch in GitHub.
```

Comment:
I kind of think how it stands gives enough scope to the person doing it to choose the method they wish to use (locally or in GitHub).

> ...first squash your commits down to a single commit [..] then merge it in GitHub

I do refer to another guide which uses the local approach, so understand that might be where the inference to do it locally comes from. If I remove that reference, I think it then becomes general enough that we're not requiring one way or the other.

On doing it in GitHub, reading the documentation does suggest that you should be able to do as required. So I say

> "Man cannot discover new oceans unless he has the courage to lose sight of the shore."
> [**André Gide**](https://en.wikipedia.org/wiki/Andr%C3%A9_Gide)

Finally on the process itself, I would wish to remain steadfast. For others to contribute I believe there will be some value in understanding how we got to where we are, and _good_ commit messages are essential to that.

I'll be honest I'd always thought of **dst-guides** within the context of developers, the idea being when someone new starts we say, go there, read it, before you

> “Go forth a conqueror and win great victories.”
> [**Virgil**](https://en.wikipedia.org/wiki/Virgil)

Again we should possibly consider making the scope and purpose of dst-guides clearer, but that's another discussion.

💭 's anyone?